charts/mission-control/templates/views/dashboard/recent-changes.yaml

```diff
@@ -51,7 +51,7 @@ spec:
     recent_changes:
       changes:
         name: "*"
-        limit: 250
+        limit: 10
```

Comment:
Must increase once we have `spec.display.table.page-size` https://github.com/flanksource/mission-control/issues/2573